builtin/game/chat.lua

```diff
@@ -904,7 +946,8 @@ core.register_chatcommand("days", {
 
 core.register_chatcommand("shutdown", {
 	params = "[<delay_in_seconds> | -1] [reconnect] [<message>]",
-	description = "Shutdown server (-1 cancels a delayed shutdown)",
+	description = "Shutdown server ("..core.colorize(core.COLOR_PARAM, "-1") ..
```

Comment:
Spaces around binary operators: `a .. b`

Same for the following changes in this function.